tests/ImageSharp.Tests/Processing/Processors/ColorMatrix/FilterTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Numerics;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing;
+using SixLabors.ImageSharp.Tests.TestUtilities.ImageComparison;
+
+using SixLabors.Primitives;
+using Xunit;
+
+namespace SixLabors.ImageSharp.Tests.Processing.Processors.Filters
+{
+    public class FilterTest : FileTestBase
+    {
+        [Theory]
+        [WithFileCollection(nameof(DefaultFiles), DefaultPixelType)]
```

Comment:
I think we should stop using `DefaultFiles` in new test cases & keep it for legacy tests only.
For filters, I suggest to test using `[WithTestPatternImages]` + probably `CalliphoraPartial.png` to have one realistic photo.

Reply:
Will update.